config/jobs/kubernetes/sig-node/crio.yaml

```diff
@@ -383,57 +396,6 @@ periodics:
     testgrid-tab-name: ci-crio-cgroupv1-node-e2e-resource-managers
```

Comment:
What about `ci-crio-cgroupv1-node-e2e-resource-managers`?

Reply:
There is a special ignition file for that one.

Reply:
So no `cluster: k8s-infra-prow-build` for this one?

Reply:
Yea due to https://github.com/kubernetes/test-infra/blob/master/jobs/e2e_node/crio/latest/image-config-cgrpv1-resource-managers.yaml